### PR TITLE
Linter fixes for line length and prettier conflicts

### PR DESCRIPTION
**Pull request by RinseV**
Prettier config was added for the linter

Also merged in ``test/general``:

- Added missing mood test case for model conversion
- Added endpoint in test cases as env variable in case we switch to the production server
- Max line length fixes for linter (sometimes)

### CODE REVIEW

**Review by alexnik99 — COMMENTED** [review]
Reviewed

File: src/services/food/foodParser.ts [hunk 1]
```diff
@@ -61,4 +61,7 @@ export enum FoodSource {
 /**
  * All possible input types for food data,
  */
-export type FoodInput = XOR<Consumptie[], XOR<AbbottData[], XOR<FoodDiaryData[], NightScoutTreatmentModel[]>>>;
+export type FoodInput = XOR<
+    Consumptie[],
+    XOR<AbbottData[], XOR<FoodDiaryData[], NightScoutTreatmentModel[]>>
+>;
```
Comment (konstantinmvelkov) [inline src/services/food/foodParser.ts:67]: Maybe a few new lines will be better

**Review by konstantinmvelkov — APPROVED** [review]

**Review by alexnik99 — APPROVED** [review]
Approved

File: src/services/food/foodParser.ts (hunk 1 above)
Comment (RinseV) [reply src/services/food/foodParser.ts:67]: Blame prettier, "Format on Save" turns it into this 😛

Comment (konstantinmvelkov) [reply src/services/food/foodParser.ts:67]: 🤷‍♂️

**Pull request merged**